app/controllers/action_plans_controller.rb

```diff
@@ -3,6 +3,9 @@ class ActionPlansController < ApplicationController
 
   def show
     @project = Project.find(params[:project_id])
+
+    ensure_unarchived!
```

Comment:
I'm wondering if we want to prevent the action plan for an archived projects, sounds like is a harmless thing to allow?

Reply:
Yes, I think we should't allow to create an action plan to archived projects, that's why this line is ensuring that the project is unarchived.